app/models/user.rb

```diff
@@ -13,6 +13,7 @@ class User < ApplicationRecord
 
   before_save :update_api_key
 
+  attribute :stories_order, :string
```

Comment:
This should be a database attribute, so I don't think this will help. This creates an in-memory attribute, but will not persist to the database, so the stories order will not change.

For some reason, it appears that the migrations haven't run on the build images when somebody is trying to upgrade their Docker image. When I try setting up an image from scratch, I don't get the error.

Reply:
Did you try running migrations from scratch?

Reply:
All I've done so far is to follow [the docker instructions](https://github.com/stringer-rss/stringer/blob/5d0b9f65a6011bc9ebc4bd5535ed5924657e404b/docs/Docker.md) from our docs with a fresh environment. Unfortunately, I haven't had time to do any deeper investigation.